gtfs_realtime_translators/translators/njt_rail.py

```diff
@@ -0,0 +1,97 @@
+import pendulum
+
+from gtfs_realtime_translators.factories import FeedMessage, TripUpdate
+import xmltodict
+
+
+class NjtRailGtfsRealtimeTranslator:
+    ROUTE_ID_LOOKUP = {
+        'atlantic_city_ine': '1',
+        'montclair-boonton_line': None,
+        'main_line': '5',
+        'bergen_county_line': '6',
+        'morristown_line': '7',
+        'gladstone_branch': '8',
+        'northeast_corridor_line': '9',
+        'north_jersey_coast_line': None,
```

Comment:
Just clarifying: Does none just mean realtime data doesn't exist for this particular route, even though the route exists in the static GTFS?

Reply:
I think I see what's going on. This value can be set to None if (1) There is no Realtime GTFS feed for the route or (2) There is a GTFS feed, but the routeID can change based on destination, so we're calculating that on [L#97](https://github.com/Intersection/gtfs-realtime-translators/pull/6/commits/ce6a6224fb0cc3cfd4c653107d20cf55e7f75c8f#diff-60762aaa1cfc79422090c3046168973fR97)

Reply:
Good question. And yes. Just to clarify, the routes indeed exist in the GTFS data, but we have to extrapolate this ID in certain cases based on their realtime data and what is truth in their GTFS data.